docs/doc/60-contributing/03-rfcs/20220809-share.md

```diff
@@ -0,0 +1,171 @@
+
+Syntax:
+
+```sql
+CREATE SHARE [IF NOT EXISTS] <share_name> [ COMMENT = '<string_literal>' ]
+```
+
+### Drop Share
+
+Remove the specified share from the system and immediately revoke access to all tenants (i.e., the tenants that created the database from that share).
+
+Syntax:
+
+```sql
+DROP SHARE [IF EXISTS] <share_name>
+```
+
+### Alter Share
+
+Modify the properties of an existing share.
+
+- Add or remove tenants from the tenant list.
+
+Syntax:
+
+```sql
+ALTER SHARE [IF EXISTS] <share_name> {ADD | REMOVE} TENANTS = <tenant_name> [, <tenant_name>, ...]
+```
+
+### Grant share object privilege TO SHARE
+
+Grant access to the database and other supported database objects (tables, views, etc.) to a share. Granting permissions to these objects effectively adds them to the share, which can then be shared with one or more tenants.
+
+Syntax:
+
+```sql
+GRANT object_privilege ON grant_share_object_name TO SHARE <share_name>
+
+object_privilege := [USAGE|SELECT|REFERENCE_USAGE]
+
+grant_share_object_name := { DATABASE <db_name> | TABLE <db_name>.<table_name> }
+```
+
+### Revoke share object privilege FROM SHARE
+
+Revoke access to databases and other supported database objects (tables, views, etc.) in the share. Revoking permissions to these objects effectively removes them from the share, making them inaccessible to all tenants that have created databases in the share.
+
+Syntax:
+
+```sql
+REVOKE object_privilege ON grant_share_object_name FROM SHARE <share_name>
+
+object_privilege := [USAGE|SELECT|REFERENCE_USAGE]
+
+grant_share_object_name := { DATABASE <db_name> | TABLE <db_name>.<table_name> }
+```
+
+### Show Grants
+
+Lists all access control privileges that have been explicitly granted to tenants and shares.
+
+Syntax:
+
+```sql
+SHOW GRANTS
+SHOW GRANTS ON grant_share_object_name
+SHOW GRANTS TO {TENANT <tenant_name> | SHARE <share_name> }
+SHOW GRANTS OF SHARE <share_name>
+
+grant_share_object_name := { DATABASE <db_name> | TABLE <db_name>.<table_name> }
+```
+
+- `SHOW GRANTS`: Syntactically equivalent to SHOW GRANTS TO TENANT current_tenant. Lists all the roles granted to the current tenant.
+- `SHOW GRANTS ON grant_share_object_name`: Lists all privileges that have been granted on the object.
+- `SHOW GRANTS TO ...`:
+  - `TENANT <tenant_name>`: Lists all the privileges granted to the tenant.
+  - `SHARE <share_name>`: Lists all the privileges granted to the share.
+- `SHOW GRANTS OF SHARE <share_name>`: Lists all the tenants for the share and indicates the tenants that are using the share.
+
+### Create Database, Table FROM Share
+
+After tenants have permission to access the share objects, can create the database from the share.
+
+Syntax:
+
+```sql
+CREATE DATABASE <name> FROM SHARE <provider_tenant>.<share_name>
+```
```

Comment:
I am not very familiar with the `share`, thus my question is what's the difference between a normal database and one created from a share?

Reply:
users have no permission to modify data from a database created from a share.